lib/defopt.py

```diff
@@ -91,6 +92,7 @@ def _ti_get_origin(tp):
         return {  # Py<3.7.
             typing.List: list,
             typing.Iterable: collections.abc.Iterable,
+            getattr(typing, "Collection", object()): collections.abc.Collection,
```

Comment:
Looks like collection.abc.Collection also needs to be replaced by `getattr(collection.abc, "Collection", object())` again for Py3.5 support (below too).  (To be clear, on Py3.5, nothing will have an "origin" of `Collection`, so we can just replace that by an arbitrary object that is never referenced anywhere else.)

Reply:
Got it, and would you prefer `getattr(collection.abc, "Collection", object())` to something like
```python
try:
    from collections.abc import Collection
except ImportError:
    Collection = object()
```
and then just using `Collection` wherever `collections.abc.Collection` is referenced?

Reply:
Actually that looks like a better solution, yes.

Reply:
Tests need to be updated too (use `defopt.Collection` instead of `typing.Collection`).

Reply:
ah yeah whoops, will do